nf_core/launch.py

```diff
@@ -208,6 +223,32 @@ def get_pipeline_schema(self):
                 log.error("Could not build pipeline schema: {}".format(e))
                 return False
 
+    def try_fetch_release_tags(self):
+        """Tries to fetch tag names of pipeline releases from github
+
+        Returns:
+            release_tags (list[str]): Returns list of release tags
+
+        Raises:
+            LookupError, if no releases were found
+        """
+        # Fetch releases from github api
+        releases_url = "https://api.github.com/repos/nf-core/{}/releases".format(self.pipeline)
```

Comment:
Need to test / streamline this behaviour a little for running offline I think.

Also need to make this work with non nf-core repositories.

Reply:
Done